cvpy/annotation/base/Task.py

```diff
@@ -1,9 +1,15 @@
+from cvpy.base.ImageTable import ImageTable
+from cvpy.annotation.base.Project import Project
+
+
 class Task(object):
 
-    def __init__(self, task_id: str, start_image_id: str, end_image_id: str) -> None:
-        self._task_id = task_id
-        self._start_image_id = start_image_id
-        self._end_image_id = end_image_id
+    def __init__(self, image_table: ImageTable, project: Project) -> None:
+        self._task_id = None
+        self._project = project
+        self._image_table = image_table
+        self._start_image_id = 0
+        self._end_image_id = image_table.table.tableinfo().TableInfo.Rows.values[0] - 1
```

Comment:
Just noting this in case anyone wonders. The "image_table" argument is required in a Task constructor and the "table" argument is required in an ImageTable constructor - so it is safe to call these methods without checking if their objects exist.